packages/adapter-typeorm/package.json

```diff
@@ -38,26 +46,27 @@
     "test:containers": "tests/test.sh",
     "test": "tests/test.sh"
   },
+  "dependencies": {
+    "@auth/core": "workspace:*"
+  },
   "devDependencies": {
     "@next-auth/adapter-test": "workspace:*",
     "@next-auth/tsconfig": "workspace:*",
     "jest": "^27.4.3",
     "mssql": "^7.2.1",
     "mysql": "^2.18.1",
-    "next-auth": "workspace:*",
     "pg": "^8.7.3",
     "sqlite3": "^5.0.8",
-    "typeorm": "0.3.7",
+    "typeorm": "0.3.15",
```

Comment:
Locked in on `0.3.15` as something is likely broken in `0.3.16`

I think https://github.com/typeorm/typeorm/pull/10041 would solve the issue.